hapi-utils/src/main/java/com/hederahashgraph/fee/CryptoFeeBuilder.java

```diff
@@ -669,9 +669,9 @@ public FeeData getCryptoDeleteLiveHashTxFeeMatrices(TransactionBody txBody, SigV
     long sbpr = 0;
 
     // calculate BPT - Total Bytes in Transaction
-    int txBodySize = 0;
+    long txBodySize = 0;
     txBodySize = getCommonTransactionBodyBytes(txBody);
-    bpt = txBodySize + getCryptoDeleteLiveHashBodyBodyTxSize() + sigValObj.getSignatureSize();
+    bpt = (long) txBodySize + getCryptoDeleteLiveHashBodyBodyTxSize() + sigValObj.getSignatureSize();
```

Comment:
same here, remove this cast

Reply:
Yes changes were in my local. Fixed it. Thanks!